app/javascript/packs/case_contact.js

```diff
@@ -0,0 +1,8 @@
+window.onload = function() {
+  var milesInput = document.getElementById("case_contact_miles_driven");
+  milesInput.onchange = function(){
+    if(milesInput.value > 0){
+      alert('Are you sure?');
```

Comment:
Thanks for working on this! This issue also needs a check for the contact type. The pop-up should only show up if the contact_type is not "in person". Please also make the message more clear about what the problem is. "Contact type is currently X, remember to double-check that is correct before submitting"